_faq/10-cdi-weld-versions.asciidoc

```diff
@@ -3,7 +3,7 @@
 Weld versions do not match CDI versioning.
 An overview is provided in the following table:
 
-[cols="5*^", options="header"]
```

Comment:
asciidoc rookie question - what does this magical `5*^` do? :)

Reply:
It means five columns with the same width and aligned horizontally: centered.

Reply:
Thanks for explanation